### PR TITLE
Updated docs to remove background icons examples;

### LINKED ISSUE #500: Replace occurrences of background icons (span tag) with foreground icons (svg tag)
Any internal dependency on the skin background icon module causes a perf hit due to the significant CSS payload it imports. Let's convert any occurrences of background icons (i.e. the span) tag to foreground icons (i.e. the svg tag).

### PR DESCRIPTION
**Pull request by seangates**
## Description
- made sure the foreground (SVG) icon was shown as "preferred" in the docs
- removed the demo app example for background icons

## Context
We want to promote foreground (SVG) icons as the way to correctly use icons with the core components. Very luckily, none of our components use the background icons. Hurray!

## References
Fixes #500

### CODE REVIEW

**Comment by seangates** [comment]
@ianmcburnie That reference is to the `<ebay-icon>` tag, not the Skin bundle for `@ebay/skin/icon`.

**Comment by ianmcburnie** [comment]
Oh yes. Oops. Seems like my brain was switched off. Thanks Sean.

**Comment by ianmcburnie** [comment]
@seangates Maybe this is the line I meant:

https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-icon/browser.json#L5

**Comment by ianmcburnie** [comment]
As discussed offline, let's target this for a 1.4.0 branch and release.

**Comment by ianmcburnie** [comment]
By the way, the intent of #500 isn't just to remove background icons from our docs, but all internal references too.

**Comment by seangates** [comment]
@ianmcburnie Yup. I just haven't gotten to your comment. Working on it now. I will create a new 1.4.0 branch and re-point my changes to that target branch.

**Comment by seangates** [comment]
`"path": "@ebay/skin/icon/foreground"` does not appear to be allowed right now.

**Comment by ianmcburnie** [comment]
Did you update to skin 6.3?

**Comment by seangates** [comment]
@ianmcburnie This should be good now, and pointed to `1.4.0`.

File: src/components/ebay-button/index.js [hunk 1]
```diff
@@ -38,22 +43,34 @@ function getInitialState(input) {
         classes.push(`${mainClass}--${priority}`);
     }
 
-    if (!isBadged && (size === 'small' || size === 'large')) {
-        classes.push(`${mainClass}--${size}`);
+    if (size === 'small' || size === 'medium' || size === 'large') {
```
Comment (ianmcburnie) [inline src/components/ebay-button/index.js]: Are these code changes intentionally part of this icon related PR?

**Comment by seangates** [comment]
@ianmcburnie Fixed!

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
   ],
   "devDependencies": {
     "@ebay/browserslist-config": "^1.0.0",
-    "@ebay/skin": "6.2.0",
+    "@ebay/skin": "6.3.0",
```
[the remark this answers is not in the document]
Comment (seangates) [reply package.json]: This is a `devDependency` though, and wouldn't be used when pulled into Inception, correct?

[the remark this answers is not in the document]
Comment (ianmcburnie) [reply package.json]: peerDependency is also pointing to 6.2.0

**Comment by seangates** [comment]
@ianmcburnie I have updated the package versions for Skin.

Also, I went ahead and updated the icon example with a lot less icons (there was literally no need), so please note that change.

**Comment by ianmcburnie** [comment]
I think we should keep at least one example of background icon.

**Comment by seangates** [comment]
There’s an example in the README, which I did not remove, and will show up in the marketplace.

**Comment by seangates** [comment]
@ianmcburnie I was able to find the additional dependency paths, and everything should be right. I no longer see the CSS file being linked in the demo app.

**Pull request merged**